### DISCUSSION TITLE
Impossible to open up Mytourbook 24.5.0.

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1350] [General]
After installing Mytourbook 24.5.0 over previous version, I get this
What's wrong?
![2024-05-17 09_09_59-Mytourbook](https://github.com/mytourbook/mytourbook/assets/4082252/a0a9067d-4f8f-4e5b-b2ab-d0d23e2140d3)
[1715930042936.log](https://github.com/mytourbook/mytourbook/files/15345126/1715930042936.log)

**Comment by GurVJ** [comment]
It seems that you are using Java version 13, but version 17 is needed.
![Capture 001_20240517](https://github.com/mytourbook/mytourbook/assets/64196685/aec0c140-f875-4f3a-b42f-60c246f9c4fd)

**Comment by gringojack** [comment]
Thank you.
Mytourbook version 24.5.0 beeing installed as well as Java version 13; where do I download Java version 17 from, and how to remove and replace Java 13 by Java 17?

**Comment by GurVJ** [comment]
Oracle Java last releases
[https://www.oracle.com/fr/java/technologies/downloads/](url)

Else,

Link from MyTourBook requirements, using Temurin 
[https://adoptium.net/fr/temurin/releases/?version=17](url)

**Comment by gringojack** [comment]
Thank you for help GurVJ

**Comment by wolfgang-ch** [comment]
Simply use MT with the embedded Java and you don't have to care about it

https://github.com/mytourbook/mytourbook/releases/download/24.5.0_2024-05-15_0536/mytourbook-24.5.0-win-setup-with-java-runtime.exe
